tests/unittests/test-extmodules.cpp

```diff
@@ -0,0 +1,104 @@
+﻿/*! @file */
+/*
+	Copyright (C) 2022, Sakura Editor Organization
+
+	This software is provided 'as-is', without any express or implied
+	warranty. In no event will the authors be held liable for any damages
+	arising from the use of this software.
+
+	Permission is granted to anyone to use this software for any purpose,
+	including commercial applications, and to alter it and redistribute it
+	freely, subject to the following restrictions:
+
+		1. The origin of this software must not be misrepresented;
+		   you must not claim that you wrote the original software.
+		   If you use this software in a product, an acknowledgment
+		   in the product documentation would be appreciated but is
+		   not required.
+
+		2. Altered source versions must be plainly marked as such,
+		   and must not be misrepresented as being the original software.
+
+		3. This notice may not be removed or altered from any source
+		   distribution.
+*/
+#include <gtest/gtest.h>
+
+#include "TExtModule.hpp"
+
+#include "extmodule/CBregexpDll2.h"
+#include "extmodule/CHtmlHelp.h"
+#include "extmodule/CIcu4cI18n.h"
+#include "extmodule/CMigemo.h"		//TSingletonなのでテストできない
```

Comment:
消し忘れ発見。

```suggestion
#include "extmodule/CMigemo.h"
```

`TSingleton`を外したのでテストできるようになっています。

ここで重大な「事実」に気付きます。。。。
「`TSingleton`なのでテストできない」が正しいとするなら、
`TSingleton`には「テストできない」という「不具合」があることになります。